pom.xml

```diff
@@ -42,8 +42,8 @@
         <version.hibernate-validator>8.0.1.Final</version.hibernate-validator>
         <version.jakarta.validation-api>3.0.2</version.jakarta.validation-api>
         <version.zeebe.spring-boot>8.5.0</version.zeebe.spring-boot>
-        <version.camunda.spring-boot>7.21.0</version.camunda.spring-boot>
-        <version.spring-boot>3.2.4</version.spring-boot>
+        <version.camunda.spring-boot>7.22.0</version.camunda.spring-boot>
+        <version.spring-boot>3.3.3</version.spring-boot>
```

Comment:
Why not 3.3.4?

Reply:
3.3.3 is hard-wired in start.camunda.com that's why I chose it. I'm checking with 3.3.4.